src/Catalog/ProcessCommitItemBatchAsync.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+
+namespace NuGet.Services.Metadata.Catalog
+{
+    public delegate Task ProcessCommitItemBatchAsync(
```

Comment:
>ProcessCommitItemBatchAsync [](start = 25, length = 27)

nit: perhaps these tiny delegate files can be rolled into CatalogCommitUtilities at the top

Reply:
I kept one type per file.  `CatalogCommitUtilities` is already fairly large.